service/r2/service.go

```diff
@@ -359,13 +364,21 @@ func (s *service) fetchRollupRuleHistory(w http.ResponseWriter, r *http.Request)
 	return s.sendResponse(w, http.StatusOK, data)
 }
 
+func (s *service) updateRuleSet(w http.ResponseWriter, r *http.Request) error {
```

Comment:
Can you move this to below `validateNamespace` to group similar methods together? Actually `validateNamespace` should probably be called `validateRuleSet` instead, can you update that as well (I know it's not your change but it looks strange :\)?

Reply:
Ping